### PR TITLE
manage account and subscription

### PR DESCRIPTION
**Pull request by kpslp**
Hi @SenthilKumarSivakami and @denisecollard . I've made changes to the doc. Do check. Thanks.

### CODE REVIEW

File: manage-techbiz-account.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+> **Note:** The Billing admin of a TechBiz account can view the account and subscription details but can't edit the details.
+
+## Prerequisites
+
+- You need a [non-SE GSIB](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/glossary) device.
+- You need to be logged in to [TechBiz](https://portal.techbiz.suite.gov.sg/) portal.
+- An approved [**TechBiz account**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/create-account).
+
+## Change subscription and billing information
```
Comment (SenthilKumarSivakami) [inline manage-techbiz-account.md:22]: Should it be "Change subscription admins and billing information"

File: manage-techbiz-account.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+> **Note:**
+> - As the admins are public officers, their TechPass ID is the same as their organisational email address.
+> - To edit other details of a subscription or billing admin, the respective administrators must update their TechPass profile from the [TechPass portal](https://portal.techpass.suite.gov.sg). For more information, see [**TechPass user guide**](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/edit-profile).
+> - If an admin is removed, they will no longer have access to the account.
+
+7. To change the SBU code, go to the **Billing information** section, and enter the new SBU code.
+8. Click **Submit**.
+
+A confirmation message is displayed.
```
Comment (SenthilKumarSivakami) [inline manage-techbiz-account.md:46]: should the user click anything on this confirmation message or is it just a message?

File: techbiz-account.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+# TechBiz account
+
+
+To subscribe to [Singapore Government Tech Stack (SGTS)](https://www.developer.tech.gov.sg/singapore-government-tech-stack/overview/index.html) products, agencies need to have an account in TechBiz.
+
+An account allows agencies to subscribe to SGTS products, add systems to the account, manage subscription plans and view billing details. An agency can have one account for multiple systems or an account for each system.
```
Comment (SenthilKumarSivakami) [inline techbiz-account.md:6]: The TechBiz account allows ....

Comment (kpslp) [reply techbiz-account.md]: changes made

**Review by denisecollard — APPROVED** [review]

**Pull request merged**